test/acceptance/features/claimForCluster.feature

```diff
@@ -0,0 +1,157 @@
+Feature: Claim for specific cluster
+
+    Scenario: Application Namespace exists
+
+        Given Primaza Cluster "primaza-main" is running
+        And Worker Cluster "primaza-worker" for "primaza-main" is running
+        And Clusters "primaza-main" and "primaza-worker" can communicate
+        And On Primaza Cluster "primaza-main", Worker "primaza-worker"'s ClusterContext secret "primaza-kw" is published
+        And On Worker Cluster "primaza-worker", application namespace "applications" exists
+        And On Worker Cluster "primaza-worker", Primaza Application Agent is deployed into namespace "applications"
```

Comment:
This step can be removed.

`On Worker Cluster "primaza-worker", application namespace "applications" exists` will create the application namespace, roles and user needed for deploying the agent, and the Cluster Environment controller will push the agent's deployment.

```suggestion
```

Reply:
The Application Agent is not getting installed in the first step.

Reply:
yes, but it will be installed by Primaza when you declare the ClusterEnvironment in the next step

Reply:
When I remove the first step, the namespace is not getting created. So the next step fails. This is why I kept both steps.

Reply:
it seems to me it's a problem with non up-to-date permissions in tests. Please try with the following changes:

```diff
diff --git a/test/acceptance/features/claimForCluster.feature b/test/acceptance/features/claimForCluster.feature
index 319b49b..5985388 100644
--- a/test/acceptance/features/claimForCluster.feature
+++ b/test/acceptance/features/claimForCluster.feature
@@ -1,3 +1,4 @@
+@wip
 Feature: Claim for specific cluster
 
     Scenario: Application Namespace exists
@@ -7,7 +8,6 @@ Feature: Claim for specific cluster
         And Clusters "primaza-main" and "primaza-worker" can communicate
         And On Primaza Cluster "primaza-main", Worker "primaza-worker"'s ClusterContext secret "primaza-kw" is published
         And On Worker Cluster "primaza-worker", application namespace "applications" exists
-        And On Worker Cluster "primaza-worker", Primaza Application Agent is deployed into namespace "applications"
         And On Primaza Cluster "primaza-main", Resource is created
         """
         apiVersion: primaza.io/v1alpha1
@@ -112,8 +112,6 @@ Feature: Claim for specific cluster
         And Worker Cluster "primaza-worker" for "primaza-main" is running
         And Clusters "primaza-main" and "primaza-worker" can communicate
         And On Primaza Cluster "primaza-main", Worker "primaza-worker"'s ClusterContext secret "primaza-kw" is published
-        And On Worker Cluster "primaza-worker", application namespace "applications" exists
-        And On Worker Cluster "primaza-worker", Primaza Application Agent is deployed into namespace "applications"
         And On Primaza Cluster "primaza-main", Resource is created
         """
         apiVersion: primaza.io/v1alpha1
diff --git a/test/acceptance/features/steps/workercluster.py b/test/acceptance/features/steps/workercluster.py
index dc1dfe7..17661b2 100644
--- a/test/acceptance/features/steps/workercluster.py
+++ b/test/acceptance/features/steps/workercluster.py
@@ -189,14 +189,18 @@ class WorkerCluster(Cluster):
 
     def __create_application_agent_identity(self, namespace: str):
         rules = [
+            client.V1PolicyRule(
+                api_groups=[""],
+                resources=["secrets"],
+                verbs=["get", "list"]),
             client.V1PolicyRule(
                 api_groups=["apps"],
                 resources=["deployments"],
                 verbs=["get", "list", "watch", "update", "patch"]),
             client.V1PolicyRule(
                 api_groups=["primaza.io"],
-                resources=["servicebindings", "serviceclaims"],
-                verbs=["get", "list", "watch"])
+                resources=["servicebindings", "servicebindings/status", "serviceclaims"],
+                verbs=["get", "list", "watch", "update", "patch"])
         ]
         self.__create_agent_identity(namespace, "agentapp", rules)
```

Reply:
Thanks! It worked.